lib/TalusTemplate.groovy

```diff
@@ -0,0 +1,48 @@
+//
+// This file holds useful functions
+// It is based on nf-core's NfcoreTemplate
+//
+
+
+class TalusTemplate {
+    //
+    // Construct and send completion email
+    //
+    // Returns
+    // -------
+    // str
+    //     The HTML email body.
+    public static String email(workflow, params) {
+
+        // Set up email variables
+        def subject = "[$workflow.manifest.name] SUCCESS: $params.experimentName"
+        if (!workflow.success) {
+            subject = "[$workflow.manifest.name] FAILED: $params.experimentName"
+        }
+
+        // Used in ../assets/email_template.html
+        def email_fields = [:]
+        email_fields['started']      = workflow.start
+        email_fields['completed']    = workflow.complete
+        email_fields['nfVersion']    = workflow.nextflow.version
+        email_fields['version']      = workflow.manifest.version
+        email_fields['runName']      = workflow.runName
+        email_fields['success']      = workflow.success
+        email_fields['dateComplete'] = workflow.complete
+        email_fields['duration']     = workflow.duration
+        email_fields['exitStatus']   = workflow.exitStatus
+        email_fields['errorMessage'] = (workflow.errorMessage ?: 'None')
+        email_fields['errorReport']  = (workflow.errorReport ?: 'None')
+        email_fields['commandLine']  = workflow.commandLine
+        email_fields['projectDir']   = workflow.projectDir
```

Comment:
It doesn't look like these variables are being used. If they are not, could they be deleted?